hist/hist/inc/THnSparse.h

```diff
@@ -210,6 +217,15 @@ class THnSparseT: public THnSparse {
               const Int_t* nbins, const Double_t* xmin = 0,
               const Double_t* xmax = 0, Int_t chunksize = 1024 * 16):
       THnSparse(name, title, dim, nbins, xmin, xmax, chunksize) {}
+   THnSparseT(const char* name, const char* title,
+              std::vector<TAxis*> axes,
+              Int_t chunksize = 1024 * 16):
+     THnSparse(name, title, axes, chunksize) {}
+   THnSparseT(const char* name, const char* title,
+             std::vector<TAxis> axes,
+             Int_t chunksize = 1024 * 16):
+     THnSparse(name, title, axes, chunksize) {}
+
```

Comment:
All of these constructors are not necessary in C++, you can just re-use the constructors of the base class:
```suggestion
template <class CONT>
class THnSparseT: public THnSparse {
 public:
   using THnSparse::THnSparse;

   TArray* GenerateArray() const override { return new CONT(GetChunkSize()); }
 private:
   ClassDefOverride(THnSparseT, 1); // Sparse n-dimensional histogram with templated content
};

```
This means that you need to make the base class constructors `public` and also put the default arguments there (in the `THnSparse` definition):
```c++
 public:
   THnSparse();
   THnSparse(const char* name, const char* title, Int_t dim,
             const Int_t* nbins, const Double_t* xmin = nullptr, const Double_t* xmax = nullptr,
             Int_t chunksize = 1024 * 16);

   ~THnSparse() override;
   ...
```
I think this is much better than having to keep these constructors in sync manually.